website/source/docs/providers/aws/r/db_instance.html.markdown

```diff
@@ -45,7 +45,7 @@ The following arguments are supported:
 * `name` - (Optional) The DB name to create. If omitted, no database is created
     initially.
 * `password` - (Required) Password for the master DB user. Note that this may
-    show up in logs, and it will be stored in the state file.
+    show up in logs, and it will be stored in the state file. Maximum 41 characters.
```

Comment:
Maybe we can just mention, that _there are limitations_ and include the link above? Eventually if you can find a better link or better way of expressing this, I'm open for suggestions.